src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -183,7 +183,7 @@ method newMessagesLoaded*(self: Module, messages: seq[MessageDto], reactions: se
         m.image,
         m.containsContactMentions(),
         m.seen,
-        m.timestamp,
+        m.whisperTimestamp,
```

Comment:
why `whisperTimestamp` instead of `timestamp`?

Reply:
> why `whisperTimestamp` instead of `timestamp`?

Whisper time should be one defined by the  mailserver, timestamp is generated on the status-go side, in case of message I am sending it could be different time set for timestamp and whisper time, while message I am receiving has the same time set for whisper time and timestamp. Now in order to avoid displaying messages in a different order for users A and B if they have sent messages to each other in the same time I think that correct way should be to use whisper timestamp.